patches/blink_file_path.patch

```diff
@@ -1,12 +0,0 @@
-diff --git a/third_party/WebKit/Source/core/fileapi/File.idl b/third_party/WebKit/Source/core/fileapi/File.idl
-index df954bc..4683b28 100644
---- a/third_party/WebKit/Source/core/fileapi/File.idl
-+++ b/third_party/WebKit/Source/core/fileapi/File.idl
-@@ -32,6 +32,7 @@
-     Exposed=(Window,Worker),
- ] interface File : Blob {
-     readonly attribute DOMString name;
-+    readonly attribute DOMString path;
```

Comment:
This patch is still needed, lots of apps (like Atom and VS Code) are relying on it.